src/sys/windows/tcp.rs

```diff
@@ -397,6 +405,10 @@ impl StreamImp {
             self.inner.socket.write_overlapped(&buf[pos..], overlapped)
         };
         match err {
+            Ok(true) if me.instant_notify => {
+                me.write = State::Empty;
+                self.add_readiness(me, Ready::writable());
```

Comment:
I think you should be able to get that through `WSAGetOverlappedResult`, right? That is currently [exposed in miow](https://docs.rs/miow/0.1.5/i686-pc-windows-msvc/miow/net/trait.TcpStreamExt.html#tymethod.result) as of recently.